specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1189 @@
+    "version" : "V3.0-Preview.1",
+    "title" : "Text Analytics API (V3.0-Preview.1)"
+  },
+  "host" : "westus.api.cognitive.microsoft.com",
+  "basePath" : "/text/analytics/V3.0-preview.1",
+  "tags" : [ ],
+  "schemes" : [ "https" ],
+  "security" : [ {
+    "apiKeyHeader" : [ ]
+  }, {
+    "apiKeyQuery" : [ ]
+  } ],
+  "paths" : {
+    "/entities" : {
+      "post" : {
+        "summary" : "Entities from open domain",
+        "description" : "The API returns a list of general named entities (\\\"Person\\\", \\\"Location\\\", \\\"Organization\\\" etc) in a given document. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking#supported-types-for-named-entity-recognition\">Supported Entity Types in Text Analytics API</a> for the list of supported Entity Types. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/text-analytics-supported-languages\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
+        "operationId" : "5ac4251d5b4ccd1554da7631",
+        "consumes" : [ "application/json", "text/json" ],
+        "produces" : [ "application/json", "text/json" ],
+        "parameters" : [ {
+          "name" : "modelVersion",
+          "in" : "query",
+          "description" : "(optional) This value indicates which model be used for scoring .",
+          "required" : false,
```

Comment:
By default, OpenAPI keeps every parameter as optional. So, adding a `required: false` on a query parameter is not a required.